Library/Homebrew/extend/ENV/std.rb

```diff
@@ -303,6 +303,10 @@ def libstdcxx
     end
   end
 
+  def java_cache
+    append "_JAVA_OPTIONS", "-Duser.home=#{HOMEBREW_CACHE}/java_cache"
+  end
```

Comment:
Heh, so there are. I can't read for crap, apparently. Made the change locally after Baptiste's comment anyhow, just hadn't pushed/tested yet so doing that at the moment.